### PR TITLE
Document new concurrency support in Ludicrous mode

### PR DESCRIPTION
**Pull request by aaroncarey**
Documentation for https://github.com/dgraph-io/dgraph/pull/6060

- Noted new `--ludicrous_concurrency` configuration setting.
- Some rewording for usage and style.

Fixes DGRAPH-2758

---
This change is [Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph/6939)

### CODE REVIEW

File: wiki/content/deploy/ludicrous-mode.md [hunk 1]
```diff
@@ -8,33 +8,41 @@ weight = 13
...
 
-Yes, ludicrous mode works with the cluster set up with multiple data shards.
\ No newline at end of file
+Yes, Ludicrous mode works with the cluster set up with multiple data shards.
+
+## How does Ludicrous mode handle concurrency?
+
+Ludicrous mode now runs mutations concurrently per predicate. This is enabled
+by default with 2000 concurrent threads available, but you can adjust the number
+of concurrent threads available using the `--ludicrous_concurrency` configuration setting on the Alpha and Zero nodes in a cluster.
```
Comment (NamanJain8) [inline wiki/content/deploy/ludicrous-mode.md:48]: This config flag only exists in `dgraph alpha`.

**Review by NamanJain8 — CHANGES_REQUESTED** [review]

**Review by aaroncarey — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph/6939)* status: 0 of 1 files reviewed, 1 unresolved discussion (waiting on @bucanero, @danielmai, @MichaelJCompton, and @NamanJain8)

___
*[wiki/content/deploy/ludicrous-mode.md, line 48 at r2](https://reviewable.io/reviews/dgraph-io/dgraph/6939#-MMcKlY0LOvHq_XlRHEa-r2-48:-MMqDOHs2f5JFnyOi3C6:b-1y8lq3) ([raw file](https://github.com/dgraph-io/dgraph/blob/7a953ef7f3049f3cbf4d61045518336fe61c430c/wiki/content/deploy/ludicrous-mode.md#L48)):*
<details><summary><i>Previously, NamanJain8 (Naman Jain) wrote…</i></summary><blockquote>

This config flag only exists in `dgraph alpha`.
</blockquote></details>

Thanks for the clarification! Fixed.

**Review by anand-chandrashekar — APPROVED** [review]
> Reviewed 1 of 1 files at r3.
> *[Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph/6939)* status: all files reviewed, 1 unresolved discussion (waiting on @bucanero, @danielmai, @MichaelJCompton, and @NamanJain8)

File: wiki/content/deploy/ludicrous-mode.md (hunk 1 above)
Comment (aaroncarey) [reply wiki/content/deploy/ludicrous-mode.md]: Fixed! Thank you 👍

**Pull request merged**